sky/jobs/server/core.py

```diff
@@ -230,6 +232,24 @@ def launch(
             controller=controller,
             task_resources=sum([list(t.resources) for t in dag.tasks], []))
 
+        consolidation_mode_job_id = None
+        if managed_job_utils.is_consolidation_mode():
+            # Create local directory for the managed job.
+            pathlib.Path(prefix).expanduser().mkdir(parents=True, exist_ok=True)
+            set_job_info_kwargs = {
+                'workspace': skypilot_config.get_active_workspace(
+                    force_user_workspace=True),
+                'entrypoint': common_utils.get_current_command(),
+            }
```

Comment:
Why do we need a kwargs dict here? Can't we just pass these directly to the function call below?